src/emm/emm_cli.py

```diff
@@ -211,5 +225,25 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100, allow_extra_args=True))
```

Comment:
Why `allow_extra_args=True`? It doesn't look like we are using the extra args.

Reply:
I think pass the revision would be treated as an extra_args, otherwise the cli would complain about unknown args.
I changed it to use
```python
@click.argument('revision', nargs=-1)
```